backend/app.js

```diff
@@ -1,21 +1,29 @@
 const express = require("express");
 const app = express();
+const bodyParser = require("body-parser");
 const port = 3001;
+const db = require("./queries");
```

Comment:
sorry for not catching this earlier, but can this be renamed to show that it's specifically for announcements?